TicTacToe/TicTacToe.cs

```diff
@@ -16,62 +16,130 @@ public static void Main()
         {
             do
             {
-                DrawBoard();
-                GetInput();
-
+            DrawBoard(); 
+            playerTurn = (playerTurn == "X") ? "O" : "X";
+            GetInput();
             } while (!CheckForWin() && !CheckForTie());
+            Console.WriteLine("Game Over!");
 
-            // leave this command at the end so your program does not close automatically
+        // leave this command at the end so it does not close automatically
             Console.ReadLine();
         }
 
         public static void GetInput()
         {
+        // Ask player to enter "row" and "collumn" coordinates to place their "X" or "O" on a spot
+        // Collect the values of the row and collumn (spot) and assign them to variables
             Console.WriteLine("Player " + playerTurn);
             Console.WriteLine("Enter Row:");
             int row = int.Parse(Console.ReadLine());
             Console.WriteLine("Enter Column:");
             int column = int.Parse(Console.ReadLine());
+        // Check to see if spot is open. If not open, choose another. Otherwise put the mark in the spot.
+        // If a new mark is placed in a spot, switch the player...X -> O, or O -> X after the mark is placed.
+            if (board[row][column] == "X" || board[row][column] == "O"){
+                Console.WriteLine("That square is already taken. Choose another:");
+                playerTurn = (playerTurn == "X") ? "O" : "X";
+            }
+            else {PlaceMark (row, column); 
+            }
         }
 
         public static void PlaceMark(int row, int column)
         {
-        // your code goes here
+        // place the playerTurn mark onto correct place on the board 
+            board[row][column] = playerTurn;
         }
 
         public static bool CheckForWin()
         {
-            // your code goes here
+        // check the three posible win methods: horizantal, vertical and diagnal for a win
 
-            return false;
+            bool anyWin = false;
+            anyWin = (HorizontalWin() == true || VerticalWin() == true || DiagonalWin() == true);
+            return anyWin;
         }
 
         public static bool CheckForTie()
         {
-            // your code goes here
-
-            return false;
+        // check to see if any spot is still open in any row
+        // if there is still an open spot game continues
+ 
+            string blank = " ";
```

Comment:
your variable name seems to be backwards.
I would have thought that open means there is a spot that is open. which mean that its not a tie yet.